Postspectator

ibrahimbayat opened the pull request:
I tried to integrate my api function to our database system. In my api I  checked if there is any user with given parameter. If there is a user with given id, I added him/her to the event of the spectator. If there is not any user with given id , I created a random person with a third party api (https://pipl.ir/v1/getPerson).  I created that user and added him/her as spectator to the given event. Functions give error if the event parameter is wrong. I tested this method with using postman application and I also tested the method with adding some tuples to the event and user table. I think the api works well and makes necessary operations.

KeremZaman commented on practice-app/app/spectator.py:
```diff
@@ -1,12 +1,14 @@
+session = Session()
 
-spectator_api = Blueprint('spectator_api', __name__)
+player_api = Blueprint('player_api', __name__)
```
This line should be the line below because it's part of the spectator API not player API.
```suggestion
spectator_api = Blueprint('spectator_api', __name__)
```

KeremZaman reviewed (changes requested):
You commited the files that you shouldn't have committed. .pyc files shouldn't be committed.
Also I suggested a change for Blueprint object.

Voursstrreds reviewed (approved):
After changing the blueprint object into the proper form, there is nothing more to advice. I could not see errors by looking. If there are such errors that cannot be seen by me can be solved in a few days.

ibrahimbayat commented:
I changed the function with the given requested changes. I forgot to delete some lines and .pyc files. While pushing to the github , I experienced some problems as always. With the help of my friend Kerem, I fixed the bugs and pushed to the branch again.
